build.py

```diff
@@ -184,7 +186,7 @@ def xcodebuild_command(self, operation, xcpretty) :
 		if (xcpretty) :
 			command += " | xcpretty"
 		if (xcpretty and operation == "test") :
-			command += " --report junit --output ./build/reports/'" + target.name + ".xml'"
+			command += " --report junit --output ./build/reports/'" + self.name + ".xml'"
```

Comment:
Just to double check that `self.name` will return the same as `target.name`, or that file name is not important (not used after).